Reworked a bit of bounty stuff.

Owai-Seek opened the pull request:
## About The Pull Request

I don't play cargo, but I do play service roles and assistant. I find it a constant hassle to have assistant and chef bounties that require botany or higher-access (or access to limited-amount items) to complete.

This makes chef bounties more accessible for a roundstart chef, without the need of botany. As "tradeoff" there is a new category for bounties that "need" both a chef, and botany to pull it off. (Let's be honest, it mainly needs botany, chef is a horrible role.)

This moves several "silly" bounties out of assistant category and makes a new bounty category for them.

## Why It's Good For The Game

More bounties are good. My pricing might be a bit off, so maybe change that a bit?

## Changelog
:cl:
add: "silly" bounties
add: "gardenchef" bounties
del: several bounties that require seriously good RNG to pull off.
tweak: moved several chef and assistant bounties to silly and gardenchef
balance: modified several bounty point rewards
server: added new files "silly.dm" and "gardenchef.dm"
/:cl:

Owai-Seek committed:
Reworked a bit of bounty stuff.

Owai-Seek committed:
Travis is mean.

Trilbyspaceclone commented on code/modules/cargo/bounties/silly.dm:
```diff
@@ -0,0 +1,43 @@
+/datum/bounty/item/silly/action_figures
+	name = "Action Figures"
+	description = "The vice president's son saw an ad for action figures on the telescreen and now he won't shut up about them. Ship some to ease his complaints."
+	reward = 4000
```
Way to high for 300 plastic 2500 would be better

Owai-Seek replied:
The reason it's so high is because you need to have they toy lathe researched (except on Delta) in order to complete it. This takes a significant amount of time and resources. This was the default number when I moved it from Assistant Bounties to Silly Bounties.

Trilbyspaceclone commented on code/modules/cargo/bounties/silly.dm:
```diff
@@ -0,0 +1,43 @@
+/datum/bounty/item/silly/clown_box
+	name = "Clown Box"
+	description = "The universe needs laughter. Stamp cardboard with a clown stamp and ship it out."
+	reward = 1000
```
800 is p-good seeings how its 1 sheet of cardboard to make this

Trilbyspaceclone commented on code/modules/cargo/bounties/assistant.dm:
```diff
@@ -76,44 +70,45 @@
-	reward = 1000
-	required_count = 2
+	description = "A famous bounty hunter is passing through the sector, and they've lost their shades. They're too embarrassed to hunt criminals with their naked eyes. Ship new sunglasses to CentCom to rectify this."
+	reward = 1750
```
Lowering this to 1000 is a good idea, its just one pare of glasses and they are easy to come by. I.e you can spawn with them...

Owai-Seek replied:
Only certain jobs spawn with sunglasses, and they're a fairly powerful item to give up for a bounty. (Flash Immunity with no vision impairment) My concern was mostly reducing the count, as getting sunglasses as an assistant can be a pain.

Trilbyspaceclone replied:
You spawn in loadout any job with sunglasses

Owai-Seek replied:
Unless you have some sort of admin or donor access that I don't access to, the only job that can spawn with "sunglasses" in the loadout are security, by spawning in with sec-huds. Otherwise, as far as I know, the QM and Bartender spawn with either sunglasses, or items that can be made into sunglasses.

Trilbyspaceclone commented on code/modules/cargo/bounties/assistant.dm:
```diff
@@ -26,8 +26,8 @@
-	reward = 1000
-	required_count = 3
+	reward = 1200
+	required_count = 3 //You can (apparently) get soap from the mining rewards vendor.
```
You can make soap out of rats

Trilbyspaceclone commented on code/modules/cargo/bounties/assistant.dm:
```diff
@@ -1,20 +1,20 @@
 	name = "Strange Object"
 	description = "Nanotrasen has taken an interest in strange objects. Find one in maint, and ship it off to CentCom right away."
-	reward = 600
+	reward = 2200
```
1000 at most, these things spawn all over the place and should not be worth as much as taser crates

Trilbyspaceclone reviewed (changes requested):
Some things are a bit high some are a bit low, -  Bolas and Metal shields should be upped a bit imo

Ghommie commented on code/modules/cargo/bounties/assistant.dm:
```diff
@@ -76,44 +70,45 @@
-	description = "A famous blues duo is passing through the sector, but they've lost their shades and they can't perform. Ship new sunglasses to CentCom to rectify this."
-	reward = 1000
-	required_count = 2
+	description = "A famous bounty hunter is passing through the sector, and they've lost their shades. They're too embarrassed to hunt criminals with their naked eyes. Ship new sunglasses to CentCom to rectify this."
```
Don't remove that Blues Brothers reference, please!

Owai-Seek replied:
I mostly want to keep assistant bounties as assistant bounties, so I could move sunglasses to Security.

Ghommie replied:
Guess I'll have to revert the description myself later, what an ordeal.

Ghommie commented on code/modules/cargo/bounties/assistant.dm:
```diff
@@ -152,14 +147,14 @@
-/datum/bounty/item/assistant/handcuffs
-	name = "Handcuffs"
+/datum/bounty/item/assistant/cuffs
+	name = "Cuffs"
```
Undo this change, I don't see a reason why an abbreviation should be used here.

Owai-Seek replied:
(As far as I know) The bounty can be completed by giving any kind of cuffs, not just handcuffs specifically. When giving out the bounty sheet, it will show as "Handcuffs", making people assume that only security handcuffs will fulfill it because they do not have access to the bounty description as cargo does.

Ghommie replied:
Yes, I know that, but they are still, by definition, handcuffs. If you want to restrict them to security handcuffs, code it.

silicons reviewed (changes requested):
what ghom said

Owai-Seek committed:
Moved some stuff around, added some more.

Moved Strange Objects and Sunglasses into silly.dm, modified a few of the bounty prices.

Owai-Seek committed:
Removed an unneeded line, added gas masks

Owai-Seek committed:
Update assistant.dm
